src/layouts/docs/server.ts

```diff
@@ -24,9 +25,13 @@ const BASE_REVALIDATE = 10
 export function getStaticGenerationFunctions({
   product,
   basePath,
+  baseName,
+  additionalRemarkPlugins = [],
 }: {
   product: Product
   basePath: string
+  baseName: string
+  additionalRemarkPlugins?: RemarkPluginEntry[]
```

Comment:
Have added an option to pass additional remark plugins, which lets us pass the `vaultUrlAdjuster` for `/vault` docs views in order to correct the `/api` link issue.